src/vc-api/README.md

```diff
@@ -0,0 +1,11 @@
+# VC Issuer API
+
+A small TypeScript library with type & [zod](https://zod.dev) definitions for the [issuer API](../../demos/vc_issuer/README.md).
+
+## Usage
+
+Build from the root directory:
+
+```bash
+$ npm run --workspace ./src/vc-api build
```

Comment:
it's not `npm run build ...` instead of `npm run .... build`?

btw. you can shorten `--workspace` with `-w` if interesting

Reply:
with `npm run build FOO`, it can be interpreted as "pass `FOO` to the build command", whereas with `npm run FOO build` it's clear that it's the build command that's being run inside the workspace

 > btw. you can shorten --workspace with -w if interesting
 
 I know :) but in scripts and READMEs I like the long form version for clarity

Reply:
> with npm run build FOO, it can be interpreted as "pass FOO to the build command", whereas with npm run FOO build it's clear that it's the build command that's being run inside the workspace

Clear is relative, I never seen `npm run -w ... build` before I have to say so it wasn't clear 😉

Reply:
Thanks for the explanation